diff -Nur pulseaudio-v16.1/src/pulsecore/creds.h pulseaudio-v16.1-mod/src/pulsecore/creds.h
--- pulseaudio-v16.1/src/pulsecore/creds.h	2022-06-21 05:54:48
+++ pulseaudio-v16.1-mod/src/pulsecore/creds.h	2023-11-08 19:55:50
@@ -34,7 +34,8 @@
 typedef struct pa_creds pa_creds;
 typedef struct pa_cmsg_ancil_data pa_cmsg_ancil_data;

-#if defined(SCM_CREDENTIALS) || defined(SCM_CREDS)
+/*  The MacOS system headers define SCM_CREDS, but not many of the related socket options. */
+#if 0

 #define HAVE_CREDS 1

